### DISCUSSION TITLE
Radioss compatibility

### DISCUSSION THREAD

**Discussion by greazee** [OpenRadioss/OpenRadioss#3182] [Q&A] [answered]
Hello, i want to write the user element for radioss, using user's library, but in examples from radioss site the element are setted up only with user's materials. So i have a question: is my element gonna be compatible with original interfaces and materials from radioss, or to use my element i need to write the new interfaces and materials by myself?

**Comment by MarianBulla** [comment, marked as answer]
Hello @greazee,
and thank you very much for your interest in OpenRadioss.
Regarding your question, yes the USER elements are not dependant on any material models. So, you can use them with the existing material models in Radioss or even without any material associated to your USER element (e.g. in case of an 1-D element).
It is fully under your control and compatible with standart and USER material models.
Which means, you don't need to write new interfaces nor your own material models for your USER elements.

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by greazee** [reply]
Good news, thank you
